examples/hello-world/src/main.rs

```diff
@@ -0,0 +1,118 @@
+extern crate serde;
```

Comment:
```suggestion
#[macro_use]
extern crate serde;
```

Reply:
I get a warning if I include this and hence removed it.

Reply:
because you weren't using `#[derive(Serialize, Deserialize)]`, but now you are because the error needs to implement these!